manifest.json

```diff
@@ -10,11 +10,13 @@
   },
   "mustUpdateAt": "2019-10-01",
   "billingOptions": {
-    "termsURL": "",
```

Comment:
https://github.com/vtex-apps/hotjar/blob/master/manifest.json#L8 Should we have terms like this here?

Reply:
Following the [billingOptions docs](https://vtex.io/docs/concepts/billing-options/), I didn't find any info about this property, so that's why I removed it.

Reply:
Following the [billingOptions docs](https://vtex.io/docs/concepts/billing-options/), I didn't find any info about this property, so that's why I removed it.